.github/PULL_REQUEST_TEMPLATE/pull_request_template.md

```diff
@@ -0,0 +1,22 @@
+### Your checklist for this pull request
+🚨 Please review the [guidelines for contributing](../../CONTRIBUTING.md) to this repository.
```

Comment:
I think we need to pump the brakes here @westeezy. 🚨 is the police car light emoji. Reviewing the guides as a pre-req is not an emergency. We don't want flashing lights to scare off future contributors.

How about 🔬 or 🍏 instead?

Reply:
🦆 please

Reply:
i like a little intimidation in my prs. 🦁

Reply:
The lion is pretty sweet too 🦁. We could use that